gp-includes/routes/translation.php

```diff
@@ -28,8 +28,15 @@ public function import_translations_get( $project_path, $locale_slug, $translati
 			return $this->die_with_404();
 		}
 
-		if ( $this->cannot_and_redirect( 'approve', 'translation-set', $translation_set->id ) ) {
-			return;
+		$can_import = $this->can( 'import', 'translation-set', $translation_set->id );
+
+		if ( ! $can_import ) {
+			$can_import_waiting = $this->can( 'import-waiting', 'translation-set', $translation_set->id );
+
+			if ( ! $can_import_waiting ) {
+				$this->redirect_with_error( __( 'You are not allowed to do that!', 'glotpress' ), $url );
+				return;
+			}
 		}
```

Comment:
My thinking is:
 - `import`: can import as current and waiting
 - `import-waiting`: can only import as waiting